src/game/game.gd

```diff
@@ -0,0 +1,19 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
+
+var characterTemplate: Resource = ResourceLoader.load("res://game/character/character.tscn")
+
+func loadMap(mapPath: String) -> void:
+	var mapToLoad: Node = ResourceLoader.load(mapPath).instance()
+	add_child_below_node(mapNode, mapToLoad)
+
+func addCharacter() -> void:
+	var newCharacter: KinematicBody2D = characterTemplate.instance()
```

Comment:
this should be going through the Characters autoload, it should look something like this:
```swift
var newCharacterResource: CharacterResource = Characters.createCharacter()
var newCharacter: KinematicBody2D = newCharacterResource.getCharacterNode()
```
This registers a new character in the characters autoload with the default network id (currently -1)

Could also remove the `characterTemplate` variable since this script shouldn't be handling the player scene

Reply:
makes sense, thanks, I was not paying attention I guess.